trie/trieStorageManager.go

```diff
@@ -406,16 +404,16 @@ func (tsm *trieStorageManager) SetCheckpoint(
 	checkpointEntry := &snapshotsQueueEntry{
 		rootHash:         rootHash,
 		mainTrieRootHash: mainTrieRootHash,
-		leavesChan:       leavesChan,
+		leavesChan:       iteratorChannels.LeavesChan,
```

Comment:
Pass the `iteratorChannels` here, instead of errChan and leavesChan

Reply:
done